src/acrogov.coffee

```diff
@@ -39,18 +42,35 @@ class AcroBot
         @loadPublicAcronyms()
         console.log("loaded public acronyms")
```

Comment:
Change all the `console.log()` statements to `@robot.logger.debug()` to reduce console noise. They'll be printed when `HUBOT_LOG_LEVEL="debug"`.